Hawkins/template/buggy_function.py

```diff
@@ -17,10 +17,13 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
         the given `angle_in_degrees`
 
     """
-    if math.floor(decimals) != decimals:
-        raise OSError('decimals should be an integer!')
+    #if math.floor(decimals) != decimals:
+    #    raise OSError('decimals should be an integer!')
+    if type(decimals) != int:
+        print('Warning : Decimals should be an integer! Converting it now (if possible)')
+        decimals = int(decimals)
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees/15
```

Comment:
Great. I'm always a bit worried about accidental integer division though (even though it can't happen in python3), so I would change 15 -> 15.0

Reply:
Good point I added 15.0 as a decimal now instead of just an integer.